src/app/components/Article/index.jsx

```diff
@@ -1,15 +1,20 @@
 import React, { Fragment } from 'react';
 import Helmet from 'react-helmet';
-import { string, node } from 'prop-types';
+import { bool, string, node } from 'prop-types';
 import Header from '../Header';
 import Footer from '../../containers/Footer';
 
-const Article = ({ lang, title, id, children }) => {
+const Article = ({ amp, lang, title, id, children }) => {
   const canonicalLink = `https://www.bbc.com/news/articles/${id}`;
+  const htmlAttributes = { lang };
+
+  if (amp) {
```

Comment:
Is it an issue if we pass an `amp: false` HTML attribute?

Reply:
If we pass in `amp: false`, we get the amp attribute on the html element as `<html amp="" ...>`. Same for `undefined` and `null`.

Reply:
The presence of an `amp` attribute on the `html` element is one of the indicators that the page is the AMP variant.  https://www.ampproject.org/docs/fundamentals/spec#required-markup I am concerned that if we have this on our canonical page, that it will indicate that it is an AMP variant instead.

Reply:
Okay. I would have accepted it if it was amp = false, but as amp="" is fairly ambiguous, let's go with the conditional logic you are implementing. Think you need to add a test for this line though.

Reply:
Because of how Helmet works, I can't carry out assertions on the `<html>` element within the Article component.  I tried with this test, which fails because of this:

```
import TestRenderer from 'react-test-renderer';
...

  it('should have an amp attribute on the html element', () => {
    const amp = true;
    const component = TestRenderer.create(
      <Article amp={amp} id={id} title={title} lang={lang} blocks={blocks} />,
    );
    const componentInstance = component.root;
    expect(componentInstance.find(el => el.type === 'html')).toHaveLength(1);
  });
```

So instead of unit tests, we have Cypress tests to capture this attribute being added.

Reply:
You should be able to do it with `enzyme`, but we are not using it yet. It's fine to be a Cypress test for now.